### DISCUSSION TITLE
pgbouncer-friendly mode: disable all use of transactions and notify?

### DISCUSSION THREAD

**Discussion by markc-versa** [procrastinate-org/procrastinate#672] [Ideas]
I'm interested in only at-most-once semantics, and I don't want to hold transactions while doing tasks. I can use an external worker notification system that's tcp-based or inotify based. Is there a way to configure the library to achieve this that's not painful?

**Comment by ewjoachim** [comment]
Procrastinate doesn't hold transaction while doing tasks. If it does, it's a bug, please report it as an issue and not a discussion.

**Comment by markc-versa** [comment]
Oh good, sorry I must have been confused while reading up on similar frameworks.
